CMakeLists.txt

```diff
@@ -290,6 +290,19 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(mips)")
     else()
         message(WARNING "The compiler does not support loongson mmi extension. NCNN_MMI will be OFF.")
     endif()
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(loongarch64)")
+    set(NCNN_TARGET_ARCH loongarch64)
```

Comment:
follow the linux kernel style https://github.com/torvalds/linux/tree/master/arch/loongarch
use loongarch instead of loongarch64 for all class names, folder names and file names